### ISSUE TITLE
Wont start on Yosemite!

### ISSUE THREAD

**Issue by crazygamedev** [stephen/airsonos#174]
Hello,
When i try and start airsonos it prints this to the screen:
```
$ airsonos
Searching for Sonos devices on network...
/usr/local/lib/node_modules/airsonos/lib/main.js:30
    devices.forEach(function(device) {
           ^
TypeError: Cannot read property 'forEach' of undefined
    at /usr/local/lib/node_modules/airsonos/lib/main.js:30:12
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/lib/logicalDevice.js:105:22
    at done (/usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:126:15)
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:32:16
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:251:17
    at done (/usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:126:15)
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:32:16
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:248:21
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/node_modules/async/lib/async.js:572:34
    at /usr/local/lib/node_modules/airsonos/node_modules/sonos/lib/events/volumeListener.js:24:14
```
I do not understand why it is doing this!
Please help me!
Regards,

Josh Selby

**Comment by davidjbradshaw** [comment]
Use Node 0.10.33 not 0.12.x

**Comment by stephen** [comment]
This has been fixed in 0.2.2
